views/faveExercise.pug

```diff
@@ -0,0 +1,19 @@
+doctype html
+html(lang="en")
+    head
+        include head
+        title FavouriteExe
```

Comment:
```suggestion
        title Favourite Exercises
```
Remember this is displayed to to the user

Reply:
Agreed. Code amended.